pkg/network/usm/testutil/grpc/client.go

```diff
@@ -150,7 +150,7 @@ type Options struct {
 
 // NewClient returns a new gRPC client
 func NewClient(addr string, options Options, withTLS bool) (Client, error) {
-	gRPCOptions := []grpc.DialOption{grpc.WithBlock()}
+	gRPCOptions := []grpc.DialOption{grpc.WithBlock()} //nolint:staticcheck // TODO (ASC) fix grpc.DialContext is deprecated
```

Comment:
Is this supposed to be here? `DialOption` is not deprecated and there's no `grpc.Dial` or `grpc.DialContext` here.

Reply:
`grpc.WithBlock` is deprecated, the message should have been concerning that instead.